txauthor.go

```diff
@@ -31,12 +31,21 @@ func (mw *MultiWallet) NewUnsignedTx(sourceWallet *Wallet, sourceAccountNumber i
 	}
 }
 
-func (tx *TxAuthor) AddSendDestination(address string, atomAmount int64, sendMax bool) {
+func (tx *TxAuthor) SetSourceAccount(accountNumber int32) {
+	tx.sendFromAccount = uint32(accountNumber)
+}
```

Comment:
These changes must have crept in from your last rebase. Please review and remove all changes NOT related to the fix this PR provides. This includes changes to your go.mod and go.sum files.